extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -185,6 +193,58 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
       );
   }
 
+  /**
+   * Builds remote V2 page attachment's UI.
+   * @private
+   */
+  buildRemoteV2_() {
+    this.setOpenThreshold_(150 /* pixels */);
+    this.setDragCap_(56 /* pixels */);
+
+    this.headerEl_.classList.add(
+      'i-amphtml-story-draggable-drawer-header-attachment-remote'
+    );
+    this.element.classList.add('i-amphtml-story-page-attachment-remote');
+
+    const link = htmlFor(this.element)`
+      <a class="i-amphtml-story-page-attachment-remote-content" target="_blank">
```

Comment:
It's not intended. Thanks for catching this. 

To prevent this we could put `pointer-events: none` on the element:
https://github.com/ampproject/amphtml/pull/33982/files#diff-23ab4d9dc3d70d03d13e3783ce2f7714ba4945fc8079aebd439dd31336dad9d6R115

And enable pointer events when in vertical rendering:
https://github.com/ampproject/amphtml/pull/33982/files#diff-83cbbde09142d0f60e6388a9a7a0bece2ea540fcad9ee699cb703c04c1cedf3fR54-R57

I don't have much context on vertical rendering and how bots are reading our pages. 
I want to be sure this wouldn't introduce any side effects.
@newmuis @gmajoulet

Reply:
This looks like it was done before the CTA button was an anchor element.
We can likely make this a div but we will need to be sure bots can crawl the CTA button.

I reverted the patch for this in this PR so we can follow up on it separately. 
I created #34015 for tracking this.